modules/exploits/linux/http/fusionpbx_php_editor_cmd_exec.rb

```diff
@@ -0,0 +1,135 @@
+####################################################################
+# This module requires Metasploit: https://metasploit.com/download #
+#  Current source: https://github.com/rapid7/metasploit-framework  #
+####################################################################
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+        "Name" => "FusionPBX =< 4.5.10 Authenticated Remote Code Execution",
```

Comment:
Module names shouldn't include a version number.  I would opt to call this something more like "FusionPBX PHPEditor Authenticated Remote Code Execution" or similar.

Reply:
Changing to your suggestion in next commit.